nucleus/deployment/common/src/main/java/fish/payara/deployment/util/GAVConvertor.java

```diff
@@ -117,7 +120,7 @@
      * the provided GAV as Strings
      */
     private Map<String, String> splitGAV(String GAV) throws MalformedURLException {
-        final String[] splitGAV = GAV.split(",|:");
+        final String[] splitGAV = GAV.split("[,:]");
         final Map<String, String> GAVMap = new HashMap<>();
         try {
             GAVMap.put("groupId", splitGAV[0].replace('.', '/'));
```

Comment:
Maven coordinates might also include `packaging` and `classifier`. If the `packaging` coordinate exists you don't have to iterate of the list of possible file extensions. If the `classifier` coordinate you won't be able to find the artefact without it.

Reply:
This may may implemented if we stick to some predefined format. In community doc defined _artifactId_, _groupId_ and _version_ only. Link in next answer

Reply:
Yes, its true according to the doc but shouldn't there be a possibility to use packaging and qualifier?

Reply:
Not allowing a user to specify packaging and qualifiers is simply a limitation / missing feature of the implementation at the moment - I'd say it's out of scope for this PR though. The use case you described makes sense, I just wouldn't bundle it into this PR to prevent scope creep, particularly since it isn't a trivial 1 line change.